tests/test_pdbx.py

```diff
@@ -29,17 +29,6 @@
 from mdtraj.formats import PDBxTrajectoryFile
 from mdtraj.testing import eq
 
-try:
-    import openmm  # noqa: F401
-
-    HAVE_OPENMM = True
-except ImportError:
-    HAVE_OPENMM = False
-
-# special pytest global to mark all tests in this module
-pytestmark = pytest.mark.skipif(not HAVE_OPENMM, reason="test_pdbx.py needs OpenMM.")
-
-
```

Comment:
In addition to the existing test for conversion, we should have a test to make sure a CIF file is read correctly. I can work on adding this - would you like me to? I'd recommend adding a CIF to `tests/data` and checking that the number of frames, atoms, unit cell, etc is correct.